components/actiongroup/index.css

```diff
@@ -79,12 +78,20 @@ governing permissions and limitations under the License.
       border-radius: 0;
       z-index: 0;
 
+      &:after {
+        --spectrum-actionbutton-focus-ring-border-radius-adjusted: 0;
```

Comment:
Honestly I'd set this on the actual element, not the pseudo-element. It seems strange to define it initially on `.spectrum-ActionButton`, but then set it in the `:after` later, even if that is where it's being used.

Reply:
It is for specificity to override the original definition.